### ISSUE TITLE
pear-shaped timeout problem

### ISSUE THREAD

**Issue by VladimirAlexiev** [shexjs/shex.js#24]
- Test URL: http://rawgit.com/shexSpec/shex.js/master/doc/shex-simple.html?schema=%3CS%3E%20%7B%20%3Cp%3E%20%5B%40aa~%5D%20%7D&data=%3Cexact%3E%20%3Cp%3E%20%22exact%22%40aa%20.%0A%3Csub%3E%20%3Cp%3E%20%22sub%22%40aa-ES%20.%0A%3CshouldFail%3E%20%3Cp%3E%20%22shouldFail%22%40aaa-ES%20.%0A&shape-map=%7BFOCUS%20%3Cp%3E%20_%7D%40%3CS%3E
- Browser: Chrome Version 60.0.3112.101 (Official Build) (64-bit)
- sometimes the example fails to load and I see "Javascript is disabled or slow to load or something went a bit pear-shaped."
- after a while, sometimes the example does load.
- once I saw some JQuery errors in the dev console
- the slowest to load is `shex-browserify.js`, 2.4Mb
![image](https://user-images.githubusercontent.com/536250/30051669-9282b2dc-922b-11e7-9948-d2a2c3074406.png)
Details:
![image](https://user-images.githubusercontent.com/536250/30051681-a0cf7c30-922b-11e7-8a49-8e27088a1459.png)
- I think I was just impatient: when I saw the empty text areas, didn't think to wait 10s for it to load
- But can its size be reduced? Or the serving speed improved?

**Comment by VladimirAlexiev** [comment]
Force-reload shows just about the same. 
![image](https://user-images.githubusercontent.com/536250/30051849-2ebb9ace-922c-11e7-831c-3c8db8d95f49.png)
`shex-browserify` is much smaller for @ericprud

**Comment by VladimirAlexiev** [comment]
One time I got status `(failed)` for `jquery.min.js` and `ShEx-bones-min.svg`, both served from w3.org.
Updated browser to Version 60.0.3112.113 (Official Build) (64-bit), no change.
Worst case `shex-browserify.js` took 34s (served from rawgit.com):
![image](https://user-images.githubusercontent.com/536250/30052120-2fd8264c-922d-11e7-853a-36d2c198ce81.png)

I think the script should just be more patient and show "Loading..." rather than an error. Then users might also become more patient :-)

**Comment by ericprud** [comment]
Yeah, it should have been more patient, but I think I've finally made it fast.
(fast like responding to an issue in just over three years).

**Comment by ericprud** [comment]
This was not a validator hang. The "…something went a bit pear-shaped" text is a **static placeholder** in the web app, shown until the JS bundle loads and replaces it; the 2017 slowness was the large browser bundle loading from the now-defunct rawgit. The app is served differently now, and the validation input itself (`<S> { <p> [@aa~] }`, a language-stem range) is instant. Closing as resolved.
